src/server.ts

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const log = require("npmlog");
+
+export function getPort(): number {
```

Comment:
👉 Reading the port form `env` isn't really a "server" responsibility. Let's move this to index for now and then introduce a "config" _class_ so that we can provide different ways of configuration e.g. per config file or view env.

Reply:
Done :)